2020/March/README.md

```diff
@@ -1,3 +1,35 @@
 # March PythonPune meetup 2020
+This was a first ever virtual event of PythonPune where everyone
+connected remotely. We had a good turn around on the YouTube live
+stream. *Read more about our live streaming setup
+[here](https://geeksocket.in/posts/live-stream-meetup-event/).*
   * [Event page on meetup.com](https://www.meetup.com/PythonPune/events/269419831/)
   * [Community Newsletter](./community_news.md)
+  * [Recording](https://youtu.be/y6W480Y_KbU)
+
+## Design Patterns and Python: Creational Patterns
+  * **Speaker**: A.P. Rajshekhar
+  * **Description**: In this talk, A.P. introduced everyone to design
+    patterns. He covered details like when to use these patterns, why
+    one should use them. This talk covered the creational patterns
+    like singleton and factory pattern. We had a nice discussion
+    around how one should design things by following these patterns
+    etc.
+
+  * [Slides]()
```

Comment:
Till the time when slides link is added, we should mention the link is not yet added. Otherwise reader think it is broken link.

How about this:
```
[Slides](/ "Link to the slide will be adding soon.. Stay tune.")
```

Reply:
Nice idea! I have made the change.